src/collections/_documentation/workflow/releases/health.md

```diff
@@ -41,6 +41,26 @@ The release details page focuses on an individual release. Elements of the relea
 
 [{% asset releases/release_details.png alt="View of the release details page showing an overview graph, files changed, and other details." %}]({% asset releases/release_details.png @path %})
 
+Hover over specific dates and times in the overview graph for a quick perspective of events that crashed, behaved abnormally, errored, or are healthy.
+
+Crashed 
+
+: The app had an explicit hard crash. You’ll likely see a corresponding Sentry issue for the crash.
+
+Abnormal 
+
+: The app went Out of Memory (OOM) and was forced to quit by the OS/timed out/froze. There’s usually no corresponding Sentry issue because this is a passive action.
+
+Errored 
+
+: The app shut down normally, but there were non-fatal errors in the session. Most likely, there are issues/errors in Sentry.
+
+Healthy 
+
+: The app shuts down (or goes into the background) normally, and there were no errors in that session.
```

Comment:
Errored and Healthy potentially are too specific as well. A session may end without the app shutting down. Especially in the new platforms outside of mobile, a session will not necessarily correspond to the application lifetime.

How about:

> The session ends normally and no errors occurred during its lifetime.